src/web/client/extension.ts

```diff
@@ -681,31 +680,3 @@ async function fetchArtemisData(orgId: string) {
     WebExtensionContext.serviceEndpointCategory = artemisResponse.stamp;
     WebExtensionContext.clusterLocation = getECSOrgLocationValue(artemisResponse.response.clusterName, artemisResponse.response.clusterNumber);
 }
-
-function logOneDSLogger(queryParamsMap: Map<string, string>) {
```

Comment:
Why are we removing this?

Reply:
we log this in the case "portal" - this is redundant - not sure why we started to log it here